Makefile

```diff
@@ -304,6 +317,12 @@ test_shortint: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --profile $(CARGO_PROFILE) \
 		--features=$(TARGET_ARCH_FEATURE),shortint,internal-keycache -p tfhe -- shortint::
 
+.PHONY: test_shortint_cov # Run the tests of the shortint module with code coverage
+test_shortint_cov: install_tarpaulin
+	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_CHECK_TOOLCHAIN) tarpaulin --profile $(CARGO_PROFILE) \
+		--out Xml --output-dir coverage/shortint --line --engine Llvm --timeout 500 \
```

Comment:
By default tarpaulin create a cobertura.xml in the current working directory. To avoid any overwrite with future layers of coverage, I decided to create a `coverage` directory that groups reports by layers.

Reply:
I mean is it in the .gitignore file ?

Reply:
Nope it's not as for now. Should I add it now since the approval test suite just begun ?

Reply:
yes please, updating the gitignore file is fine, we'll wait for https://github.com/zama-ai/tfhe-rs/pull/498/checks?check_run_id=16707790649 to complete (integer tests)